[2025-02-14T19:01:04.369+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T19:01:04.572+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_data_pipeline.fetch_stats_vlog scheduled__2025-02-13T00:00:00+00:00 [queued]>
[2025-02-14T19:01:04.683+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_data_pipeline.fetch_stats_vlog scheduled__2025-02-13T00:00:00+00:00 [queued]>
[2025-02-14T19:01:04.695+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-14T19:01:04.857+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_stats_vlog> on 2025-02-13 00:00:00+00:00
[2025-02-14T19:01:04.958+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5022) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-14T19:01:04.973+0000] {standard_task_runner.py:72} INFO - Started process 5034 to run task
[2025-02-14T19:01:04.929+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'youtube_data_pipeline', 'fetch_stats_vlog', 'scheduled__2025-02-13T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/fetch-api-statistic-bigquery.py', '--cfg-path', '/tmp/tmpyncakr6b']
[2025-02-14T19:01:05.004+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask fetch_stats_vlog
[2025-02-14T19:01:07.841+0000] {task_command.py:467} INFO - Running <TaskInstance: youtube_data_pipeline.fetch_stats_vlog scheduled__2025-02-13T00:00:00+00:00 [running]> on host 79903c5a5135
[2025-02-14T19:01:09.149+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_stats_vlog' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-13T00:00:00+00:00'
[2025-02-14T19:01:09.152+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T19:01:11.456+0000] {logging_mixin.py:190} INFO - Fetched stats for 10 videos in category vlog
[2025-02-14T19:01:11.464+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-14T19:01:11.530+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T19:01:11.532+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=youtube_data_pipeline, task_id=fetch_stats_vlog, run_id=scheduled__2025-02-13T00:00:00+00:00, execution_date=20250213T000000, start_date=20250214T190104, end_date=20250214T190111
[2025-02-14T19:01:11.846+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T19:01:11.887+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-02-14T19:01:12.158+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-14T19:01:12.210+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
